pkg/configurator/validating_webhook.go

```diff
@@ -1,416 +0,0 @@
-package configurator
```

Comment:
Could you carve removal of the configuration webhook into a separate PR. A lot of the code here will be reused for https://github.com/openservicemesh/osm/issues/3281. I'd prefer to retain this and tweak this separately.

Reply:
+1, this will also reduce the size of the current PR, making it easier to review

Reply:
validating_webhook is recovered